libs/utils/android/benchmark.py

```diff
@@ -203,9 +211,15 @@ def __init__(self):
             raise
 
         self._log.info('=== Execution...')
-        self.wl.run(out_dir=self.out_dir,
-                    collect=self._getBmCollect(),
-                    **self.bm_params)
+        for iter in range(self.bm_iterations):
```

Comment:
What I don't like here is that we are not providing a simple support to just wait between two iterations if we do not need to reboot. Why not keeping these two functionalities separated?

For the time being the patch Juri is proposing can be extended with a simple `--interations_pause` parameter to provide a simple yet useful support to inject a small [s] pause between iterations.
For most of the tests we are running now that should be just good enough.

In a second phase we can add a proper rebooting support, with timeout, which is a different concept then a pause between iterations.